src/OSPSuite.Core/Domain/Services/IPopulationRunner.cs

```diff
@@ -34,6 +34,7 @@ public interface IPopulationRunner
       /// <param name="populationData">Data table with non-table parameter values for variation</param>
       /// <param name="agingData">Data table with table parameter values for variation</param>
       /// <param name="initialValues">Data table with (molecule) initial values</param>
+      /// <param name="cancellationToken"></param>
```

Comment:
Could you add a comment here and on line 32 `simulation` where it is also blank? Even if the comment is `the simulation being run`.

Usually I'd say obvious comments should not be made, but there are some not-so-obvious parameters here already and that just triggers the warnings for the others that are very obvious.